front/app/containers/Authentication/steps/_components/TextButton.tsx

```diff
@@ -1,19 +1,21 @@
 import React from 'react';
 import styled from 'styled-components';
-import { fontSizes, colors } from 'utils/styleUtils';
+import { fontSizes } from 'utils/styleUtils';
+import { darken } from 'polished';
```

Comment:
The Polished library is something we've discussed in the FE meeting. Ideally we don't want to use Darken/Lighten (just specific colours from the component library). But given how often our designs make use of darker/lighter versions of tenant colours, we need to come up with a better system at some point (since our library colours are too restrictive) :thinking: Maybe we could just add a "TODO" comment in this file for now to mark that we're aware this should be changed??